package-lock.json

```diff
@@ -1,8 +1,4572 @@
 {
   "name": "apprentice-at",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I noticed that the `lockfileVersion` bumped up to v2 which I think is associated with the newer `npm7` instead of the earlier version used in the project. I ran into this earlier and realized it was because my nvm reset my node version back to the latest instead of node 12 used in the project. You might need to check if you're still running on node 12.0.0 and try running npm run audit one more time.